src/fusor/extract.py

```diff
@@ -0,0 +1,213 @@
+"""Module for extracting data from fusion caller output and coverting to pydantic
+objects
+"""
+
+import csv
+import logging
+from pathlib import Path
+
+from fusor.fusion_caller_models import (
+    JAFFA,
+    Arriba,
+    Cicero,
+    EnFusion,
+    FusionCatcher,
+    Genie,
+    STARFusion,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+def _check_if_file_exists(path: Path) -> bool:
+    """Check if fusions file exists
+
+    :param path: The path to the file
+    :return ``True`` if the file exists, ``False`` if not
+    """
+    if not path.exists():
+        statement = f"{path!s} does not exist"
+        _logger.error(statement)
+        return False
```

Comment:
Kind of wondering if we should be raising an error (then we won't have to return bool)